import java.util.Scanner;
public class StudentDetails {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter your name:");
        String name = sc.nextLine();
        System.out.println("Enter your age:");
        int age = sc.nextInt();
        System.out.println("Enter your marks:");
        double marks = sc.nextDouble();
        System.out.println("\n----Student Details---");
        System.out.println("Name="+name);
        System.out.println("Age="+age);
        System.out.println("Marks="+marks);

    }
}
